### ISSUE TITLE
Main Tests Results System

### ISSUE THREAD

**Issue by 7ranceaddic7** [phoronix-test-suite/phoronix-test-suite#55]
In the summary to the leftside, second section, which lists software versions installed.  OpenGL has it's version listed but not identified as 'OpenGL'.

**Comment by michaellarabel** [comment]
where/what are you talking about? An example output?  In all cases I've seen, the OpenGL version is correctly identified so would need to see your particular output to know what's wrong.

**Comment by 7ranceaddic7** [comment]
It's not that the version is not identified.  Quite the contrary, it is provided.  I have v4.4.0; it is just not labeled as OpenGl.  Sorry if that didn't come across.  I've included a screenshot (orange arrow at opengl.png) of Phoronix for you to see.  On the Main screen, presented just after launch, on the left side in the second section I have:

Ubuntu 14.10
3.16.0-31generic (x86_64)
NVIDIA 331.113
4.4.0  <-- this
GCC 4.9.1

I click on 4.4.0 and, on the next screen, Software Features has it displayed as OpenGL Driver: 4.4.0.
I've included a screenshot (orange arrow at opengl onclick.png) for this as well.  Both have an orange arrow pointing to the area in question.

Also, the kernel version (blue arrow) is presented, but not labeled on the Main screen, as well.  And is labeled elsewhere.

![orange arrow at opengl](https://cloud.githubusercontent.com/assets/4062616/6467030/82a4e7a0-c180-11e4-83ea-6ebe6ee127df.png)
![orange arrow at opengl onclick](https://cloud.githubusercontent.com/assets/4062616/6467029/82a324ce-c180-11e4-92b3-784f4c78a5ca.png)
